src/averageAges.js

```diff
@@ -15,13 +15,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let mens = people.filter(person => person.sex === 'm');
+
+  if (century !== undefined) {
+    mens = mens.filter(person => Math.ceil(person.died / 100) === century);
+  }
+
+  const totalAge = mens.reduce((sum, person) => sum
+    + (person.died - person.born), 0);
+  const averageAge = parseFloat((totalAge / mens.length).toFixed(2));
```

Comment:
The calculation of the average age is clear and well-implemented. However, consider including a comment to explain why you use `parseFloat((totalAge / mens.length).toFixed(2))` instead of simply `(totalAge / mens.length)`. This can help other developers understand your reasoning.